frontend/app/README.md

```
@@ -136,3 +136,31 @@ always pass. Can be used as a template for other tests
+3. `AppFrameworkInfo.plist`:
+    ```yaml
+    <key>CFBundleName</key>
+    <string>App Name Goes Here</string>
+    ```
+
+#### App Image
+
+Using the [flutter_launcher_icons](https://pub.dev/packages/flutter_launcher_icons) package
+
+1. Setup `pubspec.yaml`
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)